hat/assets/js/apps/Iaso/domains/forms/detail.js

```diff
@@ -219,12 +230,35 @@ const FormDetail = ({ router, params }) => {
                         <FormattedMessage {...MESSAGES.save} />
                     </Button>
                 </Box>
-                <FormVersions
-                    periodType={currentForm.period_type.value || undefined}
-                    forceRefresh={forceRefreshVersions}
-                    setForceRefresh={setForceRefreshVersions}
-                    formId={parseInt(params.formId, 10)}
-                />
+                <Box>
+                    <Tabs
+                        value={tab}
+                        classes={{
+                            root: classes.tabs,
+                        }}
+                        onChange={(_, newtab) => handleChangeTab(newtab)}
+                    >
+                        <Tab
+                            value="versions"
+                            label={formatMessage(MESSAGES.versions)}
+                        />
+                        <Tab
+                            value="attachments"
+                            label={formatMessage(MESSAGES.attachments)}
+                        />
+                    </Tabs>
+                </Box>
+                {tab === 'versions' && (
+                    <FormVersions
+                        periodType={currentForm.period_type.value || undefined}
+                        forceRefresh={forceRefreshVersions}
+                        setForceRefresh={setForceRefreshVersions}
+                        formId={parseInt(params.formId, 10)}
+                    />
+                )}
+                {tab === 'attachments' && (
+                    <FormAttachments formId={params.formId} params={params} />
```

Comment:
Is there a reason to pass the id separately (like reusing the component elsewhere)? We pass it twice since it's part of the params anyway